authy/api/resources.py

```diff
@@ -18,33 +19,40 @@
 
 
 class Resource(object):
+
     def __init__(self, api_uri, api_key):
         self.api_uri = api_uri
         self.api_key = api_key
         self.def_headers = self.__default_headers()
 
-    def post(self, path, data = {}):
+    def post(self, path, data={}):
         return self.request("POST", path, data, {'Content-Type': 'application/json'})
 
-    def get(self, path, data = {}):
+    def get(self, path, data={}):
         return self.request("GET", path, data)
 
-    def put(self, path, data = {}):
+    def put(self, path, data={}):
         return self.request("PUT", path, data, {'Content-Type': 'application/json'})
 
-    def delete(self, path, data = {}):
+    def delete(self, path, data={}):
         return self.request("DELETE", path, data)
 
-    def request(self, method, path, data = {}, headers = {}):
+    def request(self, method, path, data={}, headers={}):
         url = self.api_uri + path
-
-        params  = {}
-        headers = dict(
+        params = {}
+        temp = headers.copy()
+        if six.PY3:
+            temp = dict(
+                self.def_headers.items() |
+                headers.items()
+            )
```

Comment:
You can't add dictionaries by their items directly in python 3. The
original line was the main reason it broke for me in py3.